mri/reconstruct/linear.py

```diff
@@ -16,9 +16,11 @@
 import pysap
 from pysap.base.utils import flatten
 from pysap.base.utils import unflatten
+from modopt.signal.wavelet import get_mr_filters, filter_convolve
 
 # Third party import
-import numpy
+import numpy as np
+from joblib import Parallel, delayed
```

Comment:
Next time for the imports don't forget to have them in alphabetical order.

Reply:
So you would want them as 
```
from joblib import delayed
import numpy as np
from joblib import Parallel
```

Or just that `joblib` import must occur before `numpy`?

Reply:
well `joblib` before `numpy` (always the first letter that matters), and `modopt` before `pysap`

Reply:
cool, will change it in next commit